actions/contests.go

```diff
@@ -80,7 +80,7 @@ func ContestsCreatePost(c buffalo.Context) error {
 	// If there are no errors set a success message
 	c.Flash().Add("success", "New contest added successfully.")
 	// and redirect to the index page
-	return c.Redirect(302, "host_index")
+	return c.Redirect(302, "contests/host_index")
```

Comment:
```suggestion
	return c.Redirect(302, "host_index")
```
Works fine without this change.

Reply:
I think you made this change when I had the error on the create contest page. Are you sure its not needed?

Reply:
Yeah ik. It works without that. Commented so that we don't merge this.